pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -78,6 +82,7 @@ func newReconciler(mgr manager.Manager, clusterServiceCIDR string) (reconcile.Re
 			tracker:            vmTracker,
 			windowsVMs:         windowsVMs,
 			clusterServiceCIDR: clusterServiceCIDR,
+			recorder:           mgr.GetEventRecorderFor("wmc-controller"),
```

Comment:
Can't we use `ControllerName` here instead of `wmc-controller` and in `log`?

Reply:
You mean `wmcController`?

Reply:
No use the constant `ControllerName` everywhere we need a name.